src/backend/marsha/core/models/file.py

```diff
@@ -19,7 +19,7 @@ class UploadableFileMixin(models.Model):
         blank=True,
     )
     upload_state = models.CharField(
-        max_length=20,
+        max_length=25,
```

Comment:
I think you should think again and find a shorter machine name that sticks in 20 characters and avoid a schema migration...